options/locale/locale_en-US.ini

```
@@ -2058,6 +2066,20 @@ settings.deploy_key_desc = Deploy keys have read-only pull access to the reposit
 settings.is_writable = Enable Write Access
 settings.is_writable_info = Allow this deploy key to <strong>push</strong> to the repository.
 settings.no_deploy_keys = There are no deploy keys yet.
+settings.secrets = Secrets
+settings.pull_request_read = Pull Request Read
+settings.pull_request_read_info = "If allow pull request read the secret, it's security related."
+settings.pull_request_read_hint = Allow pull request read the secret
+settings.add_secret = Add Secret
+settings.add_secret_success = The secret '%s' has been added.
+settings.secret_value_content_placeholder = Input any content
```

Isn't that placeholder redundant?
It doesn't serve any purpose.
It would only have a purpose if it clarified for example how surounding whitespace is handled.

Speaking of which: Do we strip surrounding whitespace already? I hope so as that is in general much more user-friendly…

---

Assuming my suggestion above is implemented:
```suggestion
settings.secret_value_content_placeholder = Input any content. Whitespace at the start and end will be omitted.
```
Then suddenly this placeholder isn't redundant anymore.

---

Solved in the new PR.